sdk/dotnet/Rds/Instance.cs

```diff
@@ -489,7 +489,7 @@ public partial class Instance : global::Pulumi.CustomResource
         public Output<bool> MultiAz { get; private set; } = null!;
 
         [Output("name")]
-        public Output<string> Name { get; private set; } = null!;
+        public Output<string?> Name { get; private set; } = null!;
```

Comment:
Interesting. Benign? This might be coming out of tfgen changes actually

Reply:
This is real and correct. `name` is not set if the user correctly supplies `dbName`.